test/_init_tests.js

```diff
@@ -188,6 +188,10 @@ afterEach(() => {
     sandboxes.splice(0, sandboxes.length).forEach(sb => sb.restore());
     throw new Error('You forgot to restore your sandbox!');
   }
+  if (global.sandbox) {
+    delete global.sandbox;
```

Comment:
@jridgewell This PR is blocking some work that I'd like to merge very soon. I can remove these changes if they are too controversial. I just noticed this issue and thought it'd be good to protect from it in the future.